pkg/api/types.go

```diff
@@ -1007,6 +1007,23 @@ type PodSecurityContext struct {
 	// Use the host's ipc namespace.
 	// Optional: Default to false.
 	HostIPC bool `json:"hostIPC,omitempty"`
+
+	// Capabilities are the capabilities to add/drop when running containers
+	Capabilities *Capabilities `json:"capabilities,omitempty"`
```

Comment:
I'm starting to think we can just remove the fuzzers for these, but I'm going to double check when I re mediate the rest of the feedback.